src/codegen/object.rs

```diff
@@ -360,171 +356,108 @@ fn generate_builder(w: &mut dyn Write, env: &Env, analysis: &analysis::object::I
         analysis.name,
     )?;
     writeln!(w, "#[must_use = \"The builder must be built to be used\"]")?;
-    writeln!(w, "pub struct {}Builder {{", analysis.name)?;
+    writeln!(
+        w,
+        "pub struct {name}Builder {{
+            builder: {glib_name}::object::ObjectBuilder<'static, {name}>,
+        }}
```

Comment:
```suggestion
        "pub struct {name}Builder({glib_name}::object::ObjectBuilder<'static, {name}>);
```

maybe?

Reply:
I have plans to allow for other fields there :)